client/app/bundles/course/assessment/pages/EditPage.intl.js

```diff
@@ -0,0 +1,18 @@
+import { defineMessages } from 'react-intl';
+
+const translations = defineMessages({
+  updateAssessment: {
+    id: 'course.assessment.edit.update',
+    defaultMessage: 'Update',
+  },
+  updateSuccess: {
+    id: 'course.assessment.update.success',
+    defaultMessage: 'Assessment was updated.',
```

Comment:
https://github.com/Coursemology/coursemology2/search?utf8=✓&q=was+updated

It seems most of our code uses `was xxx` form for CRUD actions, shall we keep this?